src/test/locale/hi.js

```diff
@@ -129,6 +129,30 @@ test('format month', function (assert) {
     }
 });
 
+test('format month case', function (assert) {
+    var months = {
+            nominative: 'जनवरी_फरवरी_मार्च_अप्रैल_मई_जून_जुलाई_अगस्त_सितंबर_अक्टूबर_नवंबर_दिसंबर'.split(
+                '_'
+            ),
+            accusative: 'जनवरी_फ़रवरी_मार्च_अप्रैल_मई_जून_जुलाई_अगस्त_सितम्बर_अक्टूबर_नवम्बर_दिसम्बर'.split(
+                '_'
+            ),
+        },
+        i;
+    for (i = 0; i < 12; i++) {
+        assert.equal(
+            moment([2011, i, 1]).format('D MMMM'),
+            '१ ' + months.accusative[i],
+            '१ ' + months.accusative[i]
+        );
+        assert.equal(
+            moment([2011, i, 1]).format('MMMM'),
+            months.nominative[i],
+            '१ ' + months.nominative[i]
```

Comment:
Why does the accusative form need '१ ' but the nominative form doesn't?

Reply:
If I added day format in nominative then moment won't be able to access standalone's value. Either I can write this ( this is in the current repo): 
```
 assert.equal(
            moment([2011, i, 1]).format('MMMM'),
            months.nominative[i],
            '१ ' + months.nominative[i]
        );
```
Or I can write it like this: 
```
 assert.equal(
            moment([2011, i, 1]).format('D, MMMM'),
            '१, ' + months.nominative[i],
            '१, ' + months.nominative[i]
        );
```
But if I change it to the way you are suggesting me (I think) like this : 
```
  assert.equal(
            moment([2011, i, 1]).format('D MMMM'),
            '१ ' + months.nominative[i],
            '१ ' + months.nominative[i]
        );
```
I am merely asking moment to give me different value for the same input. Moment will give me an accusative value where as I will be checking for nominative.
I would change it to anyway you will suggest. 
PS: I think, we don't even need this test anymore because I have added the test for both inflections in parsing like you asked.

Reply:
Oh, you are right. Thanks for clarifying.